haffmpeg/sensor.py

```diff
@@ -47,7 +47,7 @@ def open_sensor(
 
         Return a coroutine.
         """
-        command = ["-vn", "-filter:a", "silencedetect=n={}dB:d=1".format(self._peak)]
+        command = ["-vn", "-filter:a", "silencedetect=n={}dB:d={}".format(self._peak, self._time_duration)]
```

Comment:
We also use the time duration for a timeout. Do you know how these two usages of time duration fit together?

Reply:
That would definitely explain the weird behavior I'm currently seeing. No matter what I set the time duration to, it always triggers the detection. Also when I'm talking for 1 second. I think we need a new parameter in this case, what do you think?

Reply:
I think so, yes. I'm not familiar with this project.

Reply:
I will do some more testing but close this for now, thanks for your feedback